src/services/Content/ContentQueries.ts

```diff
@@ -171,12 +171,13 @@ export const useDeleteContentItemMutate = (
   filterData?: FilterData,
   sortString?: string,
 ) => {
+  // Below MUST match the "useContentList" key found above or updates will fail.
```

Comment:
You might also add a comment above to make sure this gets updated too if that changes